eventually/comment/migrations/0002_comment_updated_at.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Merge migrations file to one.
0001_initial.py
002_comment_updated_at.py

output: only 0001_initial.py file.

Reply:
done